workshops/drumpad/README.md

```diff
@@ -0,0 +1,470 @@
+
+Luckily the code required to playing our sounds is minimal. Let's navigate to our `index.html` file first, and then we will add the `onclick=""` HTML attribute, and inside we call the `play(link)` with the link inside like below:
+  
+```html
+
+<div  class="pad">
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/SD0025.mp3')">A</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/SD0010.mp3')">B</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/SD0000.mp3')">C</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/RS.mp3')">D</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/OH25.mp3')">E</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/MA.mp3')">F</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/CY0010.mp3')">G</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/CH.mp3')">H</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/CB.mp3')">I</div>
+
+  <div  class="box" onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/BD0010.mp3')">J</div>
+
+  <div  class="box"  onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/BD0000.mp3')">K</div>
+
+  <div  class="box"  onclick="play('https://s3-us-west-2.amazonaws.com/s.cdpn.io/242518/CP.mp3')">L</div>
+
+</div>
+
+```
+
+And we have built the Drum Pad successfully.
+Try it out by clicking the pads and see the magic!
+
+![Image](https://media.giphy.com/media/11sBLVxNs7v6WA/giphy.gif)
+ 
+## Publishing
+
+Now that you have finished building, you should share your beautiful creation with other people! Remember, it's as easy as giving them your URL! Don't forget to share it with me on Slack @emmanuel39hanks.
+
+## Resources
+  
+- HTML Docs (https://www.w3schools.com/html/)
+- CSS Docs (https://www.w3schools.com/css/)
+- JavaScript Docs (https://www.w3schools.com/js/)
+- Audio Object (https://www.w3schools.com/JSREF/dom_obj_audio.asp)
+- Onclick event (https://www.w3schools.com/jsref/event_onclick.asp)
+- JavaScript functions (https://www.w3schools.com/js/js_functions.asp)
+
+## The End
+
+Make sure you have created an account on [repl.it](https://repl.it) to save this wonderful creation!
+
+If you face any difficulties in signing up, [watch this](https://www.youtube.com/watch?v=Mtqp4CUepk0).
+
+Now it is up to you! Do anything with this project, try to implement a way to play the drum pad with your keyboard's corresponding keys. 
+
+Check out these crazy examples!
+To finish, here are some amazing, more customized drum pad machines/beatmakers made by other developers:
+
+- A code playground on Solo Learn by Andrew Siachos (https://code.sololearn.com/WfYyBdZwc6qn/#html)
```

Comment:
Aren't you supposed to make specific changes to your code to show how you can change the version you wrote, rather than linking to entirely different versions?

Reply:
From what I know it is mainly similar projects that are extended or something like that.